src/i18n/en/translation.json

```diff
@@ -287,6 +287,7 @@
     "moderatorResetReadyStatesButton": "Reset!",
     "bannedParticipant": "Participant '{{user}}' was removed",
     "unbannedParticipant": "The ban for participant '{{user}}' was revoked",
+    "lockedBoard": "The board is currently locked; you can't make changes to it.",
```

Comment:
Maybe there should be a different toast message for participants and moderators since moderators can still make changes

Reply:
I changed it to something more general so only one text is required